h2o-bindings/bin/custom/python/gen_generic.py

```diff
@@ -13,6 +13,18 @@ def from_file(file=str):
         The imported model must be supported by H2O.
         :param file: A string containing path to the file to create the model from
         :return: H2OGenericEstimator instance representing the generic model
+
```

Comment:
Can you also fix the param and return sections in this? The output has these included in the description paragraph.